modules/serverless-install-kafka-odc.adoc

```diff
@@ -42,7 +57,7 @@ spec:
 +
 [IMPORTANT]
 ====
-To use the Kafka channel or Kafka source on your cluster, you must toggle the *Enable* switch for the options you want to use to *true*. These switches are set to *false* by default. Additionally, to use the Kafka channel, you must specify the Boostrap Servers.
+To use the Kafka channel, source, or broker on your cluster, you must toggle the *Enable* switch for the options you want to use to *true*. These switches are set to *false* by default. Additionally, to use the Kafka channel, you must specify the Bootstrap Servers.
```

Comment:
> Additionally, to use the Kafka channel, you must specify the Bootstrap Servers.

for the broker we need that too.

Background: Once we set the `bootstrapServers` on `KnativeKafka` for the broker, we can create broker objects like described in:  https://github.com/openshift/openshift-docs/pull/40701/files#diff-2b26a1f5d6a577f6180d4aa69409357cbba931d400ed7f128122602bc106aed9R18-R30

Those brokers are pointing to the configuration via their `spec.config` field. The values provided here in `KnativeKafka.Broker.DefaultConfiguration` are actually stored in the referenced `kafka-broker-config` config map

Reply:
One more note, or context here:
It is possible to leave this empty on `KnativeKafka.Broker.DefaultConfig`, but than we need to have a different configuration of our "Kafka Broker" objects.

I tried to describe that here: https://github.com/openshift/openshift-docs/blob/a64090a276803203dd83fc1854780b83bb4f4794/modules/serverless-kafka-broker-sasl-custom-config.adoc

Reply:
Updating this to say "channel or broker", but is there additional information that must be added here? The additional background / context is actually a little confusing lol